.github/workflows/release.yml

```diff
@@ -228,3 +249,113 @@ jobs:
         run: pnpm --filter miniflare test
         env:
           MINIFLARE_WORKERD_PATH: /tmp/workerd
+
+  publish-wrapper:
+    name: Publish `workerd` to NPM
+    needs: [version, upload-artifacts]
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Use Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
```

Comment:
This should be 20, but I think we should go ahead on a different PR and use `node-version-file` and make sure our bazel configuration uses that too. Right now, it's too hard to maintain the same node version in CI, bazel etc.

Reply:
Yeah agreed, as I stated in the PR description, this PR on purpose does not change any configuration option, it was 18 in the source file, I kept it 18 here as well.

Reply:
Yes definitely. I'm just leaving comments to create awareness to this problem.